6-AdvancedScenarios/2-call-api-pop/README.md

```diff
@@ -0,0 +1,343 @@
+# React single-page application calling Node.js & Express web API protected by Azure AD using Proof of Possession
```

Comment:
>React single-page application calling Node.js & Express web API protected by Azure AD using Proof of Possession [](http://example.com/codeflow?start=2&length=111)

suggestion:A React single-page application calling a Node.js & Express web API using Proof of Possession